programs/programs/nc/nc_lieap/calculator.py

```diff
@@ -0,0 +1,60 @@
+from programs.programs.calc import ProgramCalculator, Eligibility
+import programs.programs.messages as messages
+
+
+class NCLieap(ProgramCalculator):
+    expenses = ["rent", "mortgage", "heating"]
+    fpl_percent = 1.3
+    resource_limit = 2250
+    dependencies = ["income_frequency", "income_amount", "zipcode", "household_size"]
+    income_limits = {
+        # 0-50% FPL 51%-100% FPL
+        1: [816, 1632],
+        2: [1107, 2214],
+        3: [1399, 2797],
+        4: [1690, 3380],
+        5: [1981, 3963],
+        6: [2273, 4546],
+        7: [2564, 5129],
+        8: [2856, 5711],
+        9: [3147, 6294],
+        10: [3439, 6877],
+        11: [3730, 7460],
+        12: [4021, 8043],
+        13: [4313, 8626],
+        14: [4604, 9208],
+        15: [4896, 9791],
+    }
+
+    def household_eligible(self, e: Eligibility):
+        household_size = self.screen.household_size
+        gross_income = self.screen.calc_gross_income("monthly", ["all"])
+
+        # has rent or mortgage expense
+        has_rent = self.screen.has_expense(["rent"])
+        has_mortgage = self.screen.has_expense(["mortgage"])
+        e.condition(has_rent or has_mortgage)
```

Comment:
You can add multiple expenses
```suggestion
        has_rent_or_mortgage = self.screen.has_expense(["rent", "mortgage"])
        e.condition(has_rent_or_mortgage)
```